Formula/charm-tools.rb

```diff
@@ -32,11 +32,6 @@ class CharmTools < Formula
   end
 
   # Needs pip<23
-  resource "pip" do
-    url "https://files.pythonhosted.org/packages/a3/50/c4d2727b99052780aad92c7297465af5fe6eec2dbae490aa9763273ffdc1/pip-22.3.1.tar.gz"
-    sha256 "65fd48317359f3af8e593943e6ae1506b66325085ea64b706a998c6e83eeaf38"
-  end
-
```

Comment:
Looks like this is required, but not present in the packaging metadata.

Reply:
sounds like we need some upstream issue for this.